<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="UAT_Car_Sub" duration-ms="68171" started-at="2019-04-08T19:49:16Z" finished-at="2019-04-08T19:50:24Z">
    <groups>
    </groups>
    <test name="Sign Up Pass" duration-ms="27621" started-at="2019-04-08T19:49:16Z" finished-at="2019-04-08T19:49:43Z">
      <class name="suite.UAT_Sign_Up_Pass_Test">
        <test-method status="PASS" signature="setUp()[pri:0, instance:suite.UAT_Sign_Up_Pass_Test@327471b5]" name="setUp" is-config="true" duration-ms="7452" started-at="2019-04-08T19:49:16Z" finished-at="2019-04-08T19:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testToDo()[pri:0, instance:suite.UAT_Sign_Up_Pass_Test@327471b5]" name="testToDo" duration-ms="16938" started-at="2019-04-08T19:49:23Z" finished-at="2019-04-08T19:49:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testToDo -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:suite.UAT_Sign_Up_Pass_Test@327471b5]" name="tearDown" is-config="true" duration-ms="3205" started-at="2019-04-08T19:49:40Z" finished-at="2019-04-08T19:49:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- suite.UAT_Sign_Up_Pass_Test -->
    </test> <!-- Sign Up Pass -->
    <test name="Sign Up Fail" duration-ms="27707" started-at="2019-04-08T19:49:16Z" finished-at="2019-04-08T19:49:43Z">
      <class name="suite.UAT_Sign_Up_Fail_Test">
        <test-method status="PASS" signature="setUp()[pri:0, instance:suite.UAT_Sign_Up_Fail_Test@97e1986]" name="setUp" is-config="true" duration-ms="7452" started-at="2019-04-08T19:49:16Z" finished-at="2019-04-08T19:49:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testToDo()[pri:0, instance:suite.UAT_Sign_Up_Fail_Test@97e1986]" name="testToDo" duration-ms="17098" started-at="2019-04-08T19:49:23Z" finished-at="2019-04-08T19:49:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [There was an error looking up your email address!] but found [There was an error looking up your email address]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [There was an error looking up your email address!] but found [There was an error looking up your email address]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at suite.UAT_Sign_Up_Fail_Test.testToDo(UAT_Sign_Up_Fail_Test.java:97)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testToDo -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:suite.UAT_Sign_Up_Fail_Test@97e1986]" name="tearDown" is-config="true" duration-ms="3128" started-at="2019-04-08T19:49:40Z" finished-at="2019-04-08T19:49:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- suite.UAT_Sign_Up_Fail_Test -->
    </test> <!-- Sign Up Fail -->
    <test name="Sign Slider Fail" duration-ms="27751" started-at="2019-04-08T19:49:43Z" finished-at="2019-04-08T19:50:11Z">
      <class name="suite.UAT_Filter_Slider_Fail_Test">
        <test-method status="PASS" signature="setUp()[pri:0, instance:suite.UAT_Filter_Slider_Fail_Test@4590c9c3]" name="setUp" is-config="true" duration-ms="8133" started-at="2019-04-08T19:49:43Z" finished-at="2019-04-08T19:49:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testToDo()[pri:0, instance:suite.UAT_Filter_Slider_Fail_Test@4590c9c3]" name="testToDo" duration-ms="17986" started-at="2019-04-08T19:49:52Z" finished-at="2019-04-08T19:50:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Here are more vehicles available in your area] but found [Here are more vehicles available in your area:]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Here are more vehicles available in your area] but found [Here are more vehicles available in your area:]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at suite.UAT_Filter_Slider_Fail_Test.testToDo(UAT_Filter_Slider_Fail_Test.java:94)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testToDo -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:suite.UAT_Filter_Slider_Fail_Test@4590c9c3]" name="tearDown" is-config="true" duration-ms="1630" started-at="2019-04-08T19:50:10Z" finished-at="2019-04-08T19:50:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- suite.UAT_Filter_Slider_Fail_Test -->
    </test> <!-- Sign Slider Fail -->
    <test name="Sign Slider Pass" duration-ms="40447" started-at="2019-04-08T19:49:43Z" finished-at="2019-04-08T19:50:24Z">
      <class name="suite.UAT_Filter_Slider_Pass_Test">
        <test-method status="PASS" signature="setUp()[pri:0, instance:suite.UAT_Filter_Slider_Pass_Test@61a485d2]" name="setUp" is-config="true" duration-ms="14091" started-at="2019-04-08T19:49:43Z" finished-at="2019-04-08T19:49:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testToDo()[pri:0, instance:suite.UAT_Filter_Slider_Pass_Test@61a485d2]" name="testToDo" duration-ms="25204" started-at="2019-04-08T19:49:58Z" finished-at="2019-04-08T19:50:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testToDo -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:suite.UAT_Filter_Slider_Pass_Test@61a485d2]" name="tearDown" is-config="true" duration-ms="1150" started-at="2019-04-08T19:50:23Z" finished-at="2019-04-08T19:50:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- suite.UAT_Filter_Slider_Pass_Test -->
    </test> <!-- Sign Slider Pass -->
  </suite> <!-- UAT_Car_Sub -->
</testng-results>
